client/src/App.js

```diff
@@ -4,6 +4,9 @@ import Overview from './Components/ProductOverview/Overview.jsx';
 import QnaIndex from './Components/QuestionsAndAnswers/QnaIndex.jsx';
 import RatingsAndReviewsIndex from './Components/RatingsAndReviews/RatingsAndReviewsIndex.jsx'
 import RInC from './Components/RelatedItemsAndComparison/RInCIndex.jsx';
+import Navbar from './Navbar.jsx';
+import { IndexContainer } from './Navbar.jsx';
```

Comment:
I don't see this `IndexContainer` declared inside Navbar.jsx, and I also don't see it used in this file. Is it residually left here?

Reply:
Yup good catch, I left it there after trying to implement a wrapper around our App to try and center our content.